tests/snuba/api/endpoints/test_event_details.py

```diff
@@ -7,12 +7,15 @@
 from django.core.urlresolvers import reverse
 
 from sentry.models import UserReport, Group
-from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.testutils import APITestCase, SnubaTestMixin
+from sentry.testutils.skips import requires_snuba
 
 
-class EventDetailsTest(APITestCase, SnubaTestCase):
+@requires_snuba
```

Comment:
Do we need to have these decorators on tests that are in `tests/snuba`? Seems like it's already a requirement there, and we wouldn't want to skip them if snuba isn't available.

Reply:
I was hoping to merge the snuba directory in with the other tests. Having endpoint tests in two different places doesn't seem great long term. With this we could have snuba tests live with the other endpoint tests, and we could remove the separate build for snuba only tests.